tests/integration/verticalsearch.ts

```diff
@@ -22,27 +22,49 @@ const initialState: State = {
   location: {}
 };
 
-it('vertical searches set allResultsForVertical', async () => {
+const alternativeVerticals = [{
```

Comment:
nit: I usually add the corresponding typescript type to these variables just so that we know that the data is shaped correctly (same for allResultsForVertical)